pkg/pipelines/eventlisteners/eventlisteners.go

```diff
@@ -31,28 +31,28 @@ func Generate(githubRepo, ns, saName string) triggersv1.EventListener {
 					"dev-ci-build-from-pr",
 					devCIBuildFilters,
 					githubRepo,
-					"dev-ci-build-from-pr-binding",
+					"ci-build-from-pr-binding",
 					"dev-ci-build-from-pr-template",
 				),
 				createListenerTrigger(
 					"dev-cd-deploy-from-master",
 					devCDDeployFilters,
 					githubRepo,
-					"dev-cd-deploy-from-master-binding",
+					"cd-deploy-from-push-binding",
```

Comment:
And this one might be better as `github-push-binding` ?

Reply:
Renamed to `github-push-binding`